code/modules/surgery/organs/external/spines.dm

```diff
@@ -39,7 +40,7 @@
 	return GLOB.spines_list
 
 /datum/bodypart_overlay/mutant/spines/get_base_icon_state()
-	return (wagging ? "wagging" : "") + sprite_datum.icon_state //add the wagging tag if we be wagging
+	return (wagging ? "wagging_" : "") + sprite_datum.icon_state //add the wagging tag if we be wagging
```

Comment:
Need to make sure aren't other wagging spines sprites affected by this.

Reply:
does anything else really have spines?

Reply:
Looks like it's just lizards